packages/pigment-css-react/tests/generateAtomics.test.js

```diff
@@ -62,7 +65,7 @@ describe('generateAtomics', () => {
     ).to.deep.equal({
       className: 'gap--Stack-gap-lg gap--Stack-gap-xs',
       style: {
-        '--Stack-gap': 'calc(2 * 8px)',
+        '--Stack-gap-xs': 'calc(2 * 8px)',
```

Comment:
I had to bring this back because there are some variable to variable mappings, for example `--Grid-self-width` that can be assigned to `--Grid-self-width-xs`, so we need different names from the variable that points to the breakpoint variables.